monai/bundle/meta_schema.json

```diff
@@ -0,0 +1,218 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
```

Comment:
I ran the schema for existing bundles, the ones that it failed on were the new generative bundles which didn't have a `network_data_format` and so are not compliant with the current schema anyway. The open question about requiring this field or not needs resolving here, if we don't make it required then all metadata.json files pass.

Reply:
My opinion on this one is that instead of renaming and replying on the pattern property, I think these two generative models missing the `network_data_format`, so we should add them. `network_data_format` should still be a required field.
https://github.com/Project-MONAI/model-zoo/blob/e99e61aeb6fd21fc8613fd6099463ca8d7420aca/models/brats_mri_axial_slices_generative_diffusion/configs/train_diffusion.json#L18C6-L18C17
What do you think?

Reply:
I think it should stay required so there's an assumption about which network is the "main" network for a bundle and can always be found. It should be a minor tweak to the generative bundles to make them conform.

Reply:
One thing that is different that we can easily add is that `required_packages_version` and `optional_packages_version` are both required. I think existing bundles should rename `optional_packages_version` to  `required_packages_version` in their schemas if we want these properties to be required.